extensions/blocks/contact-form/editor.scss

```diff
@@ -53,13 +53,20 @@
 		margin-top:-1px;
 		margin-bottom: -3px;
 
-		.components-form-toggle {
-			margin: 2px 8px 0 0;
+		&.jetpack-field-label__required {
+			.components-form-toggle {
+				margin: 2px 8px 0 0;
+			}
+
+			.components-toggle-control__label {
+				word-break: normal;
+			}
 		}
 	}
 
 	.required {
```

Comment:
Totally unrelated to this PR but just noting that this classname is _so_ generic it must be there for a reason. 😅  Maybe to inherit something from core? Otherwise I'd expect to see `jetpack` prefix.

Reply:
It is there to inherit from core but unfortunately core has a `word-break: break-all` by default 😞

Reply:
So alternatively to this break-word fix we could just change the classname? 😀

Reply:
We would lose the core style and we would still need the `word-break` since it's done at a higher level (`.block-editor-block-list__layout .block-editor-block-list__block`).